nginx_rock/etc/main_location.conf.template

```diff
@@ -0,0 +1,8 @@
+proxy_read_timeout 300;
+proxy_pass http://main-unit:8008;
```

Comment:
where is this `main-unit` set, how is it resolved?

Reply:
I replace it in here:
https://github.com/canonical/synapse-operator/pull/268/files#diff-b9ed39bbc9c0387bd3e07da31d13373745534a1cd723d3e292c73496b12e307cR377
This template will be used for generating a new main_location.conf

Reply:
I added a comment now :)